docs/modules/sort/datastream_example.md

```diff
@@ -6,1716 +6,149 @@ sidebar_position: 3
-    ]
-}
-```
-
-- Whole-database migration example：
+Step2. Install MySQL and turn on binlog. [MySQL Installation Guide](https://dev.mysql.com/doc/mysql-installation-excerpt/5.7/en/) and [MySQL binlog](https://dev.mysql.com/doc/refman/5.7/en/replication-howto-masterbaseconfig.html)
+(You can also directly refer to Inlong MySQL Extract Node doc about MySQL turn on binlog information.)
 
-```json
-{
-  "groupId":"1",
-  "streams":[
-    {
-      "streamId":"1",
-      "nodes":[
-        {
-          "type":"mysqlExtract",
-          "id":"1",
-          "name":"mysql_input",
-          "fields":[
-            {
-              "type":"builtin",
-              "name":"data",
-              "formatInfo":{
-                "type":"string"
-              },
-              "builtinField":"MYSQL_METADATA_DATA"
-            }
-          ],
-          "properties":{
-            "append-mode":"true",
-            "migrate-all":"true"
-          },
-          "tableNames":[
-            "[\\s\\S]*.*"
-          ],
-          "hostname":"YOUR_MYSQL_HOST",
-          "username":"YOUR_USERNAME",
-          "password":"YOUR_PASSWORD",
-          "database":"[\\s\\S]*.*",
-          "port":3306,
-          "incrementalSnapshotEnabled":false
-        },
-        {
-          "type":"kafkaLoad",
-          "id":"2",
-          "name":"kafka_output",
-          "fields":[
-            {
-              "type":"base",
-              "name":"data",
-              "formatInfo":{
-                "type":"string"
-              }
-            }
-          ],
-          "fieldRelationShips":[
-            {
-              "type":"fieldRelationShip",
-              "inputField":{
-                "type":"base",
-                "name":"data",
-                "formatInfo":{
-                  "type":"string"
-                }
-              },
-              "outputField":{
-                "type":"base",
-                "name":"data",
-                "formatInfo":{
-                  "type":"string"
-                }
-              }
-            }
-          ],
-          "topic":"YOUR_TOPIC",
-          "bootstrapServers":"YOUR_KAFKA_BOOTSTRAP_SERVERS",
-          "format":{
-            "type":"csvFormat",
-            "fieldDelimiter":",",
-            "disableQuoteCharacter":true,
-            "quoteCharacter":null,
-            "allowComments":false,
-            "ignoreParseErrors":true,
-            "arrayElementDelimiter":";",
-            "escapeCharacter":null,
-            "nullLiteral":null
-          }
-        }
-      ],
-      "relations":[
-        {
-          "type":"baseRelation",
-          "inputs":[
-            "1"
-          ],
-          "outputs":[
-            "2"
-          ]
-        }
-      ]
-    }
-  ]
-}
-```
+Step3. Install Kafka and Hadoop. [Kafka Installation Guide](https://kafka.apache.org/quickstart) and [Hadoop Installation Guide](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/SingleCluster.html)
```

Comment:
It is recommended to change to "Prepare a Kafka and Hadoop environment. Please refer to[Kafka Installation Guide](https://kafka.apache.org/quickstart) or  [Hadoop Installation Guide](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/SingleCluster.html) if there is no Kafka environment or Hadoop environment."